package.json

```diff
@@ -39,18 +39,19 @@
     "babel-preset-stage-3": "^6.24.1",
     "babel-register": "^6.26.0",
     "eslint": "^5.0.1",
-    "eslint-config-airbnb": "^17.0.0",
+    "eslint-config-airbnb": "^17.1.0",
```

Comment:
Can you revert these changes?  How they are related to contract changes?

Reply:
why not include the newest versions of the library? they are working well

Reply:
ok